readme.md

```diff
@@ -154,7 +154,7 @@ $ npm install --save-dev @babel/preset-react
 Next, create a file `source.js`, where you'll type code that uses Ink:
 
 ```jsx
-import React from 'react';
+import * as React from 'react';
```

Comment:
Caught by my search&replace, but also recommended in "ESM", albeit under discussion: https://github.com/facebook/react/issues/11503#issuecomment-662647468